tests/pkg/flashdb_fal_cfg/Makefile

```diff
@@ -0,0 +1,26 @@
+include ../Makefile.pkg_common
+
+# select the MTD backend
+USEMODULE += flashdb_mtd
+# enable key-value database
+USEMODULE += flashdb_kvdb
+# enable time series database
+USEMODULE += flashdb_tsdb
+
+ifeq (,$(filter periph_rtc,$(FEATURES_PROVIDED)))
+  FEATURES_REQUIRED += periph_rtt
+  USEMODULE += rtt_rtc
+  USEMODULE += ztimer_no_periph_rtt
+else
+  FEATURES_REQUIRED += periph_rtc
+endif
```

Comment:
This won't work unfortunately.
You don't have the `FEATURES_PROVIDED` information at this point, so you will always select `periph_rtt`

Reply:
This is the known issue with the make dependency resolution...  Maybe `DEFAULT_MODULE` can somehow be used but that only gets resolved on the last iteration.

Reply:
I did not figure out how to resolve this with `DEFAULT_MODULE` but `FEATURES_REQUIRED_ANY` seems to do a good job here.

When I compile with `BOARD=same54-xpro make` it compiles fine.
And when I remove `FEATURES_PROVIDED += periph_rtc` from `boards/same54-xpro/Makefile.features`it also compiles fine with `"Using real-time-timer RTC"`.